diff -rupN solarus-ff90dd0f3bbf2029867888b1a4cc0da06d016f26.orig/cmake/AddDependencies.cmake solarus-ff90dd0f3bbf2029867888b1a4cc0da06d016f26/cmake/AddDependencies.cmake
--- solarus-ff90dd0f3bbf2029867888b1a4cc0da06d016f26.orig/cmake/AddDependencies.cmake	2020-12-16 19:12:49.716488907 -0500
+++ solarus-ff90dd0f3bbf2029867888b1a4cc0da06d016f26/cmake/AddDependencies.cmake	2020-12-16 20:25:55.006657481 -0500
@@ -11,10 +11,6 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DI
 find_package(SDL2 "2.0.6" REQUIRED)
 find_package(SDL2_image REQUIRED)
 find_package(SDL2_ttf REQUIRED)
-find_package(OpenGL)
-if(NOT OPENGL_FOUND)
-  find_package(OpenGLES2 REQUIRED)
-endif()
 find_package(GLM REQUIRED)
 find_package(OpenAL REQUIRED)
 find_package(Vorbis REQUIRED)
@@ -22,11 +18,7 @@ find_package(VorbisFile REQUIRED)
 find_package(Ogg REQUIRED)
 find_package(ModPlug REQUIRED)
 find_package(PhysFS REQUIRED)
-if(SOLARUS_USE_LUAJIT)
-  find_package(LuaJit REQUIRED)
-else()
-  find_package(Lua "5.1" EXACT REQUIRED)
-endif()
+find_package(Lua "5.1" EXACT REQUIRED)
 
 # Explicit link to libdl is needed for Lua on some systems.
 find_library(DL_LIBRARY dl)
Binary files solarus-ff90dd0f3bbf2029867888b1a4cc0da06d016f26.orig/.git/index and solarus-ff90dd0f3bbf2029867888b1a4cc0da06d016f26/.git/index differ
